### DISCUSSION TITLE
I can't see my approved solution!

### DISCUSSION THREAD

**Discussion by BitemNet** [codewars/codewars.com#2524] [Q&A] [answered]
Can someone explain why I cannot see my solution even though it has been approved? It has happened to me many times. Has anyone had a similar problem? Is there any way I can add this solved task to the list of solved tasks, because submitting again does not give the expected results - the solved task still does not appear in the list.  

Thanks for any helpful answers :)

**Comment by ejini6969** [comment, marked as answer]
Refer [1598](https://github.com/codewars/codewars.com/issues/1598)
